### PR TITLE
Fixes #35365 - Deprecation warning for oVirt integration

### PR DESCRIPTION
**Pull request by stejskalleos**
Show warning on following pages:
* Create compute resource
* View compute resource
* Create host

### CODE REVIEW

File: app/views/compute_resources/form/_ovirt.html.erb [hunk 1]
```diff
@@ -1,3 +1,13 @@
+<div class="clearfix">
+  <div class="form-group">
+    <div class="col-md-6">
+      <div class='alert alert-warning'>
+        <%= _('oVirt integration is deprecated, more information') %>
+        <%= link_to(_('here'), 'https://access.redhat.com/announcements/6960518', target: '_blank') %>.
```
Comment (ekohl) [inline app/views/compute_resources/form/_ovirt.html.erb]: It's odd that this mentions oVirt in the header but then links to the RHV deprecation. https://ovirt.org/ doesn't mention anything about deprecation.

Also, this is not the proper way to translate. If there's a language where `here` isn't the last word, the sentence becomes weird. You should interpolate, though that's always tricky with links.

Perhaps split up the lines like this:
```suggestion
        <%= _('oVirt integration is deprecated.') %>
        <%= link_to(_('More information.'), 'https://access.redhat.com/announcements/6960518', target: '_blank') %>.
```

Comment (stejskalleos) [reply app/views/compute_resources/form/_ovirt.html.erb]: Alert updated.

> It's odd that this mentions oVirt in the header but then links to the RHV deprecation.

I agree but except the red hat announcement there is nothing I can link to, no other source of the information.

File: app/views/compute_resources/show/_ovirt.html.erb [hunk 2]
```diff
@@ -14,3 +14,13 @@
   <td><%= _("Default VNC Keyboard") %></td>
   <td><%= @compute_resource.keyboard_layout %></td>
 </tr>
+<tr>
+  <td colspan="2">
+    <div class='alert alert-warning'>
+      <%= _('oVirt integration is deprecated.') %>
```
Comment (Ron-Lavi) [inline app/views/compute_resources/show/_ovirt.html.erb]: I found this post in Ovirt blog: https://blogs.ovirt.org/2022/02/future-of-ovirt-february-2022/

> Last year, Red Hat published an update to the RHV product life cycle, announcing its end of life in August 2026, with  OpenShift Virtualization as its successor. But as a commercial software product, RHV has a different prioritization from oVirt, which is and will always be a community project. To be clear, **while there will be an End of Life for RHV, there is no planned End of Life for oVirt.**

therefore I guess we should display the warning banner only for RHV compute resource.
can we also describe more, e.g: 
```on August 31, 2022, Red Hat Virtualization entered the Maintenance Support Phase. The Maintenance Support Phase runs until August 31, 2024, followed by the Extended Life Phase, which runs until August 31, 2026,``` [Read more](https://access.redhat.com/announcements/6960518)

File: app/views/compute_resources/form/_ovirt.html.erb [hunk 3]
```diff
@@ -1,3 +1,15 @@
+<div class="clearfix">
+  <div class="form-group">
+    <div class="col-md-6">
```
Comment (Ron-Lavi) [inline app/views/compute_resources/form/_ovirt.html.erb]: is it possible to align the warning with the inputs start?
Screenshot - 2022-09-11T145344 248

**Comment by stejskalleos** [comment]
@rlavi formatting and texts updated.

Now displaying only if `foreman_satellite_theme` plugin is present

**Comment by ekohl** [comment]
> Now displaying only if `foreman_satellite_theme` plugin is present

Just wondering: aren't we taking the wrong approach here? Shouldn't `foreman_satellite_theme` display the deprecation then? Normally speaking we avoid doing downstream-only things in upstream.

**Comment by stejskalleos** [comment]
> > Now displaying only if `foreman_satellite_theme` plugin is present
> 
> Just wondering: aren't we taking the wrong approach here? Shouldn't `foreman_satellite_theme` display the deprecation then? Normally speaking we avoid doing downstream-only things in upstream.

I've been thinking same, but the only other way how to do it that came up to my mind was overriding following templates from the plugin:
```
app/views/compute_resources/form/_ovirt.html.erb
app/views/compute_resources/show/_ovirt.html.erb
app/views/compute_resources_vms/form/ovirt/_base.html.erb
```
That means having templates in two places, and that is not better than having `foreman_satellite_theme` check in the code IMO.

**Comment by ekohl** [comment]
We do use [deface](https://github.com/spree/deface#readme) to modify templates in various places. It may not be the prettiest solution, but a common one in plugins.

**Comment by stejskalleos** [comment]
Let's do it in downstream

**Pull request closed without merging**